gitgud/test_user.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from gitgud.__main__ import GitGud
+
+def test_assert_initialized():
+    pass
+def test_handle_commit():
+    pass
+def test_handle_contrib():
+    pass
+def test_handle_debug():
+    pass
+def test_handle_explain():
+    pass
+def test_handle_goal():
+    pass
+def test_handle_help():
+    pass
+def test_handle_init():
+    pass
+def test_handle_issues():
+    pass
+def test_handle_level():
+    pass
+def test_handle_levels():
+    pass
+def test_handle_load():
+    pass
+def test_handle_reset():
+    pass
+def test_handle_show():
+    pass
+def test_handle_show_tree():
+    pass
+def test_handle_skills():
+    pass
+def test_handle_solution():
+    pass
+def test_handle_status():
+    pass
+def test_handle_test():
```

Comment:
Load up a level and make sure that it it doesn't fail either when the level has been completed or when it hasn't

Also make sure it doesn't fail if we're not in a git dir